.eslintrc

```diff
@@ -12,6 +12,7 @@
   },
   "parser": "babel-eslint",
   "rules": {
+    "arrow-body-style": ["off", "always"],
```

Comment:
I'd generally say do this separately, especially since we had some discussion in an issue about it. Do you mind splitting this out into its own commit before you merge?